A14-channelz.md

```diff
@@ -728,8 +736,39 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+// A trace event is an interesting thing that happened to a channel or
+// subchannel, such as creation, address resolution, subchannel creation, etc.
+message ChannelTraceEvent {
+  // High level description of the event.
+  string description = 1;
+  // Status/error associated with this event. 
```

Comment:
Can this be absent?   If present, will it ever be Status OK?

Reply:
Done